tests/unit/utils/test_modules.py

```diff
@@ -34,3 +36,22 @@ def test_ResidualBlock_shortcut(double_module):
         double_module, shortcut=utils.ResidualBlock(double_module)
     )
     assert model(input) == ptu.approx(5.0)
+
+
+def test_SequentialWithOutChannels(subtests):
```

Comment:
Could you add another test checking whether `SequentialWithOutChannels` behaves just like an `nn.Sequential`?